src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.spec.ts

```diff
@@ -293,12 +293,11 @@ describe('RbdFormComponent', () => {
   });
 
   describe('test image configuration component', () => {
-    it('is visible', () => {
```

Comment:
can we toggle the `Advanced` accordion so that the component will be visible?

Reply:
sure, let me do that!